core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/parser/SigningMetadataModule.java

```diff
@@ -36,6 +37,8 @@ public SigningMetadataModule() {
 
     addDeserializer(Bytes.class, new BytesDeserialiser());
     addSerializer(Bytes.class, new BytesSerializer());
+
+    addSerializer(Path.class, new NioPathSerializer());
```

Comment:
NioPathSerializer is already [a built in jackson class](https://fasterxml.github.io/jackson-databind/javadoc/2.13/com/fasterxml/jackson/databind/ext/NioPathSerializer.html) that does the opposite of what you're doing here (i.e. it does `path.toUri().toString()`).

Can use the built in [`ToStringSerializer`](https://fasterxml.github.io/jackson-databind/javadoc/2.13/com/fasterxml/jackson/databind/ser/std/ToStringSerializer.html) I think...
```suggestion
    addSerializer(Path.class, new ToStringSerializer());
```

Reply:
Thats a good idea.